src/test/scala/mesosphere/mesos/ResourceMatcherTest.scala

```diff
@@ -24,7 +26,13 @@ import org.apache.mesos.Protos.Attribute
 import org.apache.mesos.{ Protos => Mesos }
 import org.scalatest.Inside
 
+import scala.concurrent.duration.Duration
+
 class ResourceMatcherTest extends UnitTest with Inside {
+
+  implicit val clock = Clock.systemUTC()
```

Comment:
does this really need to be declared for the whole file, or can it be moved into the couple of test cases where it's needed? we normally use a fake clock for tests, so this seems like an exceptional case that should be appropriately scoped

Reply:
This test is not operating with a fixture and have a bunch of those global vals. I filed a ticket to replace with a fixture https://jira.mesosphere.com/browse/MARATHON-7797